src/docs/PCA/PCAUserGuide.adoc

```diff
@@ -35,23 +35,32 @@ include::references/HDFS-SPARK.adoc[]
 
 include::references/Storm.adoc[]
 
-=== Zookeeper
+=== Kafka
 
-image::PCA_zookeeper.png[align=center]
+The Kafka service allows to deploy through ProActive Cloud Automation (PCA) Portal a Kafka broker. 
```

Comment:
It could be good to explain that Kafka needs a Zookeeper instance to be executed or that it deploy a Zookeeper if it doesn't found one.